app/particle/Particle.py

```diff
@@ -82,7 +83,7 @@ def update_velocity(self, particles):
                 if euclidean_distance == 0:
                     force = max_force
                 else:
-                    force = params.force * (1 * (distances[i])/euclidean_distance**3)/self.mass
+                    force = params.force * distances[i] / euclidean_distance**3 / self.mass
```

Comment:
While it doesn't make a difference at runtime, adding parenthesis here would make the code more readable.  

Also, Python groups left to right when handling precedence for multiplication and division; dividing something that is the quotient of a product is equivalent to multiplying the quotient by the aforementioned product.